generator/app.go

```diff
@@ -516,6 +554,15 @@ func main() {
 		}
 	}
 
+	if envVal, ok := os.LookupEnv("ANNUAL_REPORT"); ok && envVal == "true" {
```

Comment:
Would be good to add `make ANNUAL_REPORT` to the generator docs - https://github.com/kubernetes/community/blob/master/generator/README.md#how-to-use

Reply:
This could be `--annual-report-issue`